client/common/cmdline.c

```diff
@@ -1288,6 +1288,18 @@ static BOOL ends_with(const char* str, const char* ext)
 
 	return strncmp(&str[strLen - extLen], ext, extLen) == 0;
 }
+
+static void activate_smartcard_logon_rdp(rdpSettings* settings)
+{
+	settings->SmartcardLogon = TRUE;
+	settings->RdpSecurity = TRUE;
+	settings->TlsSecurity = FALSE;
```

Comment:
Just curious, isn't smartcard authentication possible with TLS security ?

Reply:
@hardening Hmm, right. Should also be possible, will check.